inst/safetyGraphics_app/modules/main/main.R

```diff
@@ -0,0 +1,143 @@
+# Server code for safetyGraphics App
```

Comment:
Seems like some code from the config module is duplicated here? Is this intentional? If so, should we create reusable functions that can be shared between the files?

Reply:
The "main" module is leftover from my previous branch! We can get rid of it.